mne/simulation/evoked.py

```diff
@@ -107,9 +107,65 @@ def simulate_noise_evoked(evoked, cov, iir_filter=None, random_state=None):
     .. versionadded:: 0.10.0
     """
     noise = evoked.copy()
-    noise.data = _generate_noise(evoked.info, cov, iir_filter, random_state,
-                                 evoked.data.shape[1])[0]
-    return noise
+    noise.data[:] = 0
+    return _add_noise(noise, cov, iir_filter, random_state)
```

Comment:
I would suggest to deprecate then simulate_noise_evoked as it's purpose was to add noise after simulating evoked data